jena-elephas/jena-elephas-mapreduce/src/main/java/org/apache/jena/hadoop/rdf/mapreduce/filter/AbstractNodeTupleFilterMapper.java

```diff
@@ -28,9 +28,7 @@
 /**
  * Abstract mapper implementation which helps in filtering tuples from the
  * input, derived implementations provide an implementation of the
- * {@link #accepts(TKey, T)}
- * 
- * 
+ * {@link #accepts(Object, AbstractNodeTupleWritable)}
```

Comment:
This just looks wrong-- the method below clearly is `accepts(TKey key, T tuple)`, and the concrete class isn't there at all.

Reply:
I agree... it was strange when it turned out that this seemed to be the fix. Without the code augmentation, I was unable to generate the Javadocs for the project. It should be noted that Javadoc generation is activated by default during an ```install``` target.